### ISSUE TITLE
Current fmt dosn't build with UWP as platform

### ISSUE THREAD

**Issue by Acretock** [fmtlib/fmt#2628]
On 8.0.1 builds normally 

Build with cmake with this
`cmake -G "Visual Studio 16 2019" -DCMAKE_SYSTEM_NAME=WindowsStore -DCMAKE_SYSTEM_VERSION=10.0 -DCMAKE_BUILD_TYPE=Release -A ARM64 -DUWP=TRUE ..`
and 
`cmake --build . --config Release --target Project`

error code is 
```cpp
fmt/chrono.h(884,5): error C3861: '_tzset':
identifier not found project\build\Project.vcxproj]
```

**Comment by Acretock** [comment]
i found out that `_tzset` is not supported in UWP, [post](https://social.msdn.microsoft.com/Forums/sqlserver/en-US/f5c99f2a-912e-4489-bf9f-3b74e609b18d/uwpwin32-functions-in-c-libraries-build-for-uwp-arm?forum=wpdevelop)

**Comment by vitaut** [comment]
Should be fixed by #2633.
